src/client/src/pages/overview/boreholeTable/bottomBarContainer.tsx

```diff
@@ -88,18 +94,37 @@ const BottomBarContainer = ({
     setIsBusy(false);
   };
 
-  const onExportMultiple = async () => {
+  const handleExportMultipleJson = async () => {
     const paginatedResponse = await getAllBoreholes(selectionModel, 1, selectionModel.length);
     const jsonString = JSON.stringify(paginatedResponse.boreholes, null, 2);
-    const blob = new Blob([jsonString], { type: "application/json" });
-    const url = URL.createObjectURL(blob);
-    const link = document.createElement("a");
-    link.href = url;
-    link.download = `bulkexport_${new Date().toISOString().split("T")[0]}.json`;
-    document.body.appendChild(link);
-    link.click();
-    link.remove();
-    URL.revokeObjectURL(url);
+    downloadData(jsonString, `bulkexport_${new Date().toISOString().split("T")[0]}.json`, "application/json");
```

Comment:
PP: `bulkexport_${new Date().toISOString().split("T")[0]}` in Funktion mit aussagekräftigen Namen auslagern.